src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor

```diff
@@ -2,8 +2,13 @@
 @inject IJSRuntime JS
 @implements IDialogContentComponent
 
-<FluentRadioGroup TValue="string" Label="Theme" Value="@_currentSetting" ValueChanged="SettingChangedAsync" Orientation="Orientation.Vertical">
-    <FluentRadio Value="@ThemeSettingSystem">System</FluentRadio>
-    <FluentRadio Value="@ThemeSettingLight">Light</FluentRadio>
-    <FluentRadio Value="@ThemeSettingDark">Dark</FluentRadio>
-</FluentRadioGroup>
+<FluentStack Orientation="Orientation.Vertical" Style="display: contents
+">
```

Comment:
errant line break here within the Style attribute?

Reply:
fixed